docs/guides/getting-started.md

```diff
@@ -1,111 +1,44 @@
 # Getting Started
 
-This guide goes step by step to getting a project running.
+## Project Setup
 
-__Warning:__ The CLI is currently only supported on OSX, though we plan to add support for Windows and Linux soon. For the steps that ask to use the CLI we will provide alternatives.
+### Using Carbon Factory
 
-* Go to a directory where you keep your projects:
+The quickest way to get started with Carbon is to use [Carbon Factory](https://github.com/sage/carbon-factory).
 
-```bash
-cd ~/development
-```
-
-* Use the CLI to scaffold a Carbon/React/Flux project (or [download a zip](https://github.com/Sage/carbon-factory/raw/master/docs/carbon-app.zip)):
+Carbon Factory provides pre-configured Webpack config and Jest config to get going with a new application in minutes.
 
-```bash
-carbon app sampleapp
-```
+### Without Using Carbon Factory
 
-* Go into your new project:
+Carbon can also be used with your own build configurations.
 
-```bash
-cd sampleapp
-```
+#### Webpack
 
-* Install the project's modules:
+We recommend you install the [Parcelify Loader](https://www.npmjs.com/package/parcelify-loader). Our components integrate with the loader to easily serve CSS assets for each component you use.
 
-```bash
-npm install
-```
+#### Browserify
 
-* Run webpack to recompile assets when file changes are made - it will then continue to run and listen for any file changes, triggering a recompile when a file updates.
+We recommend you install the [Parcelify plugin](https://www.npmjs.com/package/parcelify). Our components integrate with the loader to easily serve CSS assets for each component you use.
 
-```bash
-npm start
-```
+#### Other
 
-* Open a new terminal window, and return to the same directory:
+Alternatively you can manually import the style assets for each component you use. We are working to improve how this is managed in a future version to make it easier for everyone to get started with Carbon.
 
-```bash
-cd ~/development/sampleapp
-```
+## Application Setup
 
-* Use the CLI to create a new component (or create the new file manually in `src/components/foobar/foobar.js`):
+Install Carbon into your project: `npm install carbon-react --save`.
```

Comment:
I don't think you need to add `--save` anymore - `npm install` adds the package to `dependencies` by default now.